<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>windowNo</class>
 <widget class="QDialog" name="windowNo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>American Typewriter</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>What are you looking for in your future career?</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>631</width>
     <height>86</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="Button_Helping">
      <property name="text">
       <string>Helping people</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_earnings">
      <property name="text">
       <string>Good earnings </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_hours">
      <property name="text">
       <string>Flexible hours</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="info2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>531</width>
     <height>151</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;“Life doesn’t always present you with the perfect opportunity at the perfect time. Opportunities come when you least expect them, or when you’re not ready for them. Rarely are opportunities presented to you in the perfect way, in a nice little box with a yellow bow on top. Opportunities, the good ones, they’re messy and confusing and hard to recognize. They’re risky. They challenge you.”&lt;/p&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;-Susan Wojcicki, CEO of YouTube&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next2Button">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>350</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>CodeGirls</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TryIt">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>350</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Try it</string>
   </property>
  </widget>
  <widget class="QLabel" name="picture1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="pixmap">
    <pixmap>../Desktop/1.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>backButton</zorder>
  <zorder>label_3</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>info2</zorder>
  <zorder>next2Button</zorder>
  <zorder>TryIt</zorder>
  <zorder>picture1</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
